go.mod

```diff
@@ -1,50 +1,52 @@
 module github.com/kastenhq/kubestr
 
-go 1.21
+go 1.21.3
 
 replace github.com/graymeta/stow => github.com/kastenhq/stow v0.1.2-kasten
 
 require (
 	github.com/briandowns/spinner v1.23.0
 	github.com/frankban/quicktest v1.14.6
 	github.com/golang/mock v1.6.0
-	github.com/kanisterio/kanister v0.0.0-20230711223016-1ed0c605326b
+	github.com/kanisterio/kanister v0.0.0-20240103071146-f463367d0afe
```

Comment:
We can move to the latest version - v0.0.0-20240111010256-05ebcf95d03a

Reply:
Done